src/components/inputs/money-input/money-input.js

```diff
@@ -598,33 +609,37 @@ const MoneyInput = props => {
             {...filterDataAttributes(props)}
           />
           {props.hasHighPrecisionBadge && isHighPrecision && (
-            <div
-              css={() =>
-                getHighPrecisionWrapperStyles({
-                  isDisabled: props.isDisabled,
-                })
-              }
-            >
-              <Tooltip
-                off={props.isDisabled}
-                placement="top-end"
-                // we use negative margin to make up for the padding in the Tooltip Wrapper
-                // so that the tooltip is flush with the component
-                styles={{
-                  body: {
-                    margin: `${vars.spacingS} -${vars.spacingXs} ${vars.spacingS} 0`,
-                  },
-                }}
-                title={intl.formatMessage(messages.highPrecision)}
-                components={{
-                  WrapperComponent: TooltipWrapper,
-                }}
+            <React.Fragment>
+              <div id={`portal-${props.id}`} />
```

Comment:
Nit: `getPortalId` and `getPortalNode`.

Reply:
[870d224254e53267f7463e6f9a1d1a90b545a298](https://github.com/commercetools/ui-kit/pull/1029/commits/870d224254e53267f7463e6f9a1d1a90b545a298)